README.md

```diff
@@ -26,7 +26,24 @@ The lists below are not comprehensive: feel free to [start a discussion](https:/
 
 ### Partially supported features
 
-- `process.chdir`. This runner uses Node.js workers, that don't support `process.chdir()`. It provides a simple polyfill so that `process.chdir()` calls still affect the `process.cwd()` result, but they won't affect all the other Node.js API (such as `fs.*` or `path.resolve`).
+- `process.chdir`. This runner uses Node.js workers by default, that don't support `process.chdir()`. It provides a simple polyfill so that `process.chdir()` calls still affect the `process.cwd()` result, but they won't affect all the other Node.js API (such as `fs.*` or `path.resolve`), alternatively you can:
+
+  - Create a runner runs in child process
+
+    ```js
+    // jest-light-process-runner.js
+    import { createRunner } from "jest-light-runner";
+
+    export default createRunner({ runtime: "child_process" });
+    ```
+
+    ```diff
+    -  runner: "jest-light-runner",
+    +  runner: "./path/to/jest-light-process-runner.js",
+    ```
```

Comment:
I tested Jest support `runner: "jest-light-runner/process-runner",`, which looks much better. :)

Reply:
Of course it should work, but I wasn't sure others need it or not, this solution is fine for Prettier to use.

Question didn't get answered in https://github.com/nicolo-ribaudo/jest-light-runner/issues/86

> I don't know how to tell jest-light-runner to accept options, but it won't be a problem, we can export a factory to accept options.